src/main/resources/application.yaml

```diff
@@ -15,12 +15,14 @@ management:
 spring:
   jpa:
     database-platform: org.hibernate.dialect.PostgreSQLDialect
-    show-sql: false
+    show-sql: true
     properties:
       hibernate:
         jdbc:
           lob:
             non_contextual_creation: true
+        batch_size: 2
```

Comment:
batch size we need test multiple size based on load we except. I fell 5 would be. good.

Reply:
yes. this is added as a initial value we can change it to later as per volume

Reply:
updated to 5